src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/AuditService.kt

```diff
@@ -289,24 +289,4 @@ class AuditService(
       changes = changes,
```

Comment:
is there an optimisation we can do here - now we're pretty much always attempting to load up the staff member could we change this to take a nullable staff record rather than doing the look up again here? 

(the places where we're not passing a staff record would then be a bit more obvious as we'd haver to explicitly pass null through) 

If that seems like something worth doing we could do it as a separate follow up PR? (but wouldn't necessarily want this going live/being tested with out it)

Reply:
Good spot Andy. Yup that would definitely help - just making the changes and will add to this PR